docs/api-reference/next/image.md

```diff
@@ -11,7 +11,11 @@ description: Enable Image Optimization with the built-in Image component.
   </ul>
 </details>
 
-> Before moving forward, we recommend you to read [Image Optimization](/docs/basic-features/image-optimization.md) first.
+## Image Optimization
```

Comment:
@styfle This title will create a top result for a search on `Image Optimization`, that we may not need:

![image](https://user-images.githubusercontent.com/4278345/98055835-65b91000-1e0c-11eb-9eb1-bb3f51f6fa08.png)

If I can predict the Algolia indexing properly, it will be the second result, but it won't be a very useful result.

Reply:
I removed the h2, thanks!